.circleci/config.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/node:13
```

Comment:
you should run this on **multiple** nodejs versions startnig from the lowest one your library targets.
(Probably nodejs 10)